.github/workflows/lib-check.yaml

```diff
@@ -26,6 +26,16 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+      - name: Set up environment
+        run: |
+          pipx install tox
+          pipx install poetry
+          pipx inject poetry poetry-plugin-export
+          # TODO: Remove after https://github.com/python-poetry/poetry/pull/5980 is closed
+          poetry config warnings.export false
+      - name: Generate requirements
+        run: |
+          tox run -e generate-requirements
```

Comment:
Charmcraft complains if there are no requirements.txt when fetching libs. Maybe we should move this to the dpw?

Reply:
Touch doesn't seem to work locally for me. Possibly because PGB has `charm-binary-python-packages`.

Reply:
That will rebuild the charm again in the lib-check workflow.

Reply:
Tried it, but it moves the requirements.txt on exit: https://github.com/canonical/pgbouncer-k8s-operator/actions/runs/7449186139/job/20265256079?pr=168